canvas_modules/common-canvas/src/common-properties/controls/expression/expression.jsx

```diff
@@ -82,6 +82,16 @@ class ExpressionControl extends React.Component {
 
 	componentDidMount() {
 		this.createCodeMirrorEditor();
+
+		/*
+		 * Remove aria-label from codemirror placeholder to resolve below accessibility violation
+		 * The ARIA attributes "aria-label" are not valid for the element <span> with implicit ARIA
+		 * role "generic"
+		*/
+		const placeholderElement = document.querySelector(".properties-expression-editor-wrapper .cm-placeholder");
```

Comment:
What if there are more then 1 expression builders in the DOM?  Are we wanting to remove it from all of them?

Reply:
Yes. Codemirror is adding `aria-label` to a span element (non-interactive element).

`<span class="cm-placeholder" aria-label="placeholder Enter an SQL expression" contenteditable="false" style="pointer-events: none;">Enter an SQL expression</span>`

Reply:
Instead of removing the `aria-label` we can add role like below to resolve violation.
`placeholderElement.setAttribute("role", "note");`

Reply:
What happens when there is more then 1 expression editor?  Wouldn't this return an array?